packages/rebber-plugins/__tests__/__snapshots__/rebber.test.js.snap

```diff
@@ -145,19 +145,19 @@ a code without lang
 `;
 
 exports[`code+caption 1`] = `
-"\\\\begin{CodeBlock}{python}
+"\\\\begin{CodeBlock}[]{python}
 print('bla')
-\\\\captionof{listing}{With Source}
 \\\\end{CodeBlock}
+\\\\captionof{listing}{With Source}
 
 
 
 \\\\begin{CodeBlock}[1,2]{python}
```

Comment:
No :(

Either you put `\\\\begin{CodeBlock}[][1,2]{python}` (so the highlighting remains the second argument) or `\\\\begin{CodeBlock}{python}` because you don't need to specify highlighting.

Reply:
Fixed!

Reply:
Yep. That should do :)